ceph_iscsi_config/common.py

```diff
@@ -152,6 +153,16 @@ def _get_ceph_config(self):
 
         return cfg_dict
 
+    def init_config(self):
+        ioctx = self.ceph.cluster.open_ioctx(self.pool)
+        try:
+            ioctx.stat(self.config_name)
```

Comment:
I think you should be using this op to avoid races where a daemon and overwrite another daemon's config [1]

[1] https://github.com/ceph/ceph/blob/master/src/pybind/rados/rados.pyx#L1920

Reply:
ah! great -- I was looking for something like `create` but didn't find the equivalent python call for `IoCtx`.